2016-06-10 13:52:41.681 -0600	START		Running CalculatorTesting.feature
2016-06-10 13:52:42.188 -0600	imagefound	Calculator	found at (563, 224)	2	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:42.274 -0600	click		at (563, 224)	6	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:42.275 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 13:52:42.275 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 13:52:43.515 -0600	click	CalcClearButton	at (627, 368)	2	on_there_is_no_current_value_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:43.516 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 13:52:43.517 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 13:52:44.241 -0600	typetext		4[return]	3	on_i_enter_num___num_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:44.242 -0600	LogSuccess		I enter 2 + 2 in the calculator
2016-06-10 13:52:44.242 -0600	LogSuccess		I enter 2 + 2 in the calculator
2016-06-10 13:52:45.910 -0600	readtext		(529,264,725,320)
2016-06-10 13:52:45.911 -0600	LogSuccess		value is correct	6	on_the_value_returned_should_be_4	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:45.913 -0600	LogSuccess		the value returned should be 4
2016-06-10 13:52:45.914 -0600	LogSuccess		the value returned should be 4
2016-06-10 13:52:46.347 -0600	imagefound	Calculator	found at (563, 224)	2	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:46.436 -0600	click		at (563, 224)	6	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:46.436 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 13:52:46.437 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 13:52:47.736 -0600	click	CalcClearButton	at (627, 368)	2	on_there_is_no_current_value_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:47.737 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 13:52:47.738 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 13:52:48.476 -0600	typetext		-3[return]	3	on_i_enter_num_minus_num_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 13:52:48.477 -0600	LogSuccess		I enter 7 minus 4 in the calculator
2016-06-10 13:52:48.478 -0600	LogSuccess		I enter 7 minus 4 in the calculator
2016-06-10 13:52:48.479 -0600	LogError		the value returned should be 6
2016-06-10 13:52:48.480 -0600	LogSuccess		the value returned should be 6
2016-06-10 13:52:48.481 -0600	EndTestCase		(Duration:"6.797252", Errors:"2", Exceptions:"0", StartTime:"2016-06-10 13:52:41 -0600", Successes:"16", TestCase:"CalculatorTesting.feature", Warnings:"0")
2016-06-10 13:52:48.794 -0600	FAILURE	Screen_Error.png	2 scenarios (1 failed, 1 passed) 8 steps (1 failed, 0 skipped, 7 passed) Execution Time 0:00:06
